README.md

```diff
@@ -124,8 +124,29 @@ mostly.share(g | sd | c, email)
 mostly.unshare(g | sd | c, email)
 
 g.shares()
+g.share(email)
```

Comment:
let's just have a single line to demo syntax, i.e.
`g.share(email, "VIEW" | "EDIT" | "ADMIN")`

and drop the other 3 g.share() examples.

plus lets drop `mostly.share` and `mostly.unshare` from docs and from interface. this is the old deprecated interface.

Reply:
and please remove the `share_public` and `unshare_public` from the README, and comment out in the code. This is atm just for Admins, and we will want to rename that also in the Public API.